custom/enikshay/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from dimagi.ext.couchdbkit import Document, StringProperty, IntegerProperty
+from corehq.util.decorators import serial_task
+
+
+# option 1
+# store a SQL model mapping user_id to pk (nikshay id)
+# The nice thing about this approach is that django and postgres handle all the concurrency issues for us!
+# also, this makes it easy to look up a user by nikshay id
+# the downside is that it's a db row for every user.
```

Comment:
This doesn't bother me - I still expect this table to be small enough that its size won't matter.